src/utils/sentry.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import * as Sentry from '@sentry/react';
+import {
+  createRoutesFromChildren,
+  matchRoutes,
+  useLocation,
+  useNavigationType
+} from 'react-router-dom';
+import { SENTRY_DSN_KEY } from '@/constants/environment';
+
+Sentry.init({
```

Comment:
p4;
배포된 url 에서만 에러를 수집하고 로컬 개발 환경에서는 수집하지 않도록 Sentry init 에 대해 .env 같은 설정을 한번 더 두는 건 어떨까요?!


아래처럼 설졍하면 저희 .env.local 쪽에만 추가해주면 될 거 같아용
```suggestion
IS_DEV = process.env.IS_DEV === 'dev'
if(!IS_DEV)
Sentry.init({
```

Reply:
녀무 좋습니당~

Reply:
[Vercel에서 제공하는 환경변수](https://vercel.com/docs/projects/environment-variables/system-environment-variables) 및 node에서 기본적으로 제공하는 환경변수 사용하여 특정 환경에서 sentry가 init될 수 있도록 바꾸어보았어요~ 
https://github.com/GDSC-Hongik/gdsc-client/pull/81/commits/329519446cc8b20500d52ad6624e3ddaf32eb14a